torchrl/data/tensor_specs.py

```diff
@@ -1810,7 +1814,9 @@ def to_numpy(self, val: torch.Tensor, safe: bool = None) -> np.ndarray:
             return np.array(vals).reshape(tuple(val.shape))
         return val
 
-    def enumerate(self) -> torch.Tensor:
+    def enumerate(self, use_mask: bool = False) -> torch.Tensor:
+        if use_mask:
+            raise NotImplementedError
```

Comment:
I left this unimplemented for OneHot for now, just because I want to focus on getting MCTS working. I'll submit a followup PR in the near future to add it--or if you'd rather I just add it to this PR, I'm happy to do that